tsfile/src/main/java/org/apache/iotdb/tsfile/fileSystem/fileInputFactory/HDFSInputFactory.java

```diff
@@ -17,36 +17,22 @@
  * under the License.
  */
 
-package org.apache.iotdb.tsfile.fileSystem;
+package org.apache.iotdb.tsfile.fileSystem.fileInputFactory;
 
-import java.io.IOException;
 import java.lang.reflect.InvocationTargetException;
-import java.nio.file.Paths;
-import org.apache.iotdb.tsfile.common.conf.TSFileDescriptor;
-import org.apache.iotdb.tsfile.read.reader.DefaultTsFileInput;
 import org.apache.iotdb.tsfile.read.reader.TsFileInput;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public enum FileInputFactory {
+public class HDFSInputFactory implements FileInputFactory {
 
-  INSTANCE;
-
-  private static FSType fsType = TSFileDescriptor.getInstance().getConfig().getTSFileStorageFs();
-  private static final Logger logger = LoggerFactory.getLogger(FileInputFactory.class);
+  private static final Logger logger = LoggerFactory.getLogger(HDFSInputFactory.class);
   private static Class<?> clazz;
```

Comment:
Save the constructor is better.

Reply:
Sorry for the inconsideration! I have fixed it.